pins/drivers.py

```diff
@@ -25,12 +25,15 @@ def _assert_is_pandas_df(x, file_type: str) -> None:
 def load_path(meta, path_to_version):
     # Check that only a single file name was given
     fnames = [meta.file] if isinstance(meta.file, str) else meta.file
+
+    type = meta.type
```

Comment:
```suggestion
    _type = meta.type
```

I would slightly prefer this, so as not to be confusing with the builtin `type`.

Reply:
Definitely.